OpenEdXMobile/res/values/strings.xml

```diff
@@ -189,6 +193,8 @@
     <string name="last_accessed">Last Accessed</string>
     <!-- Label shown when no courseware content is available -->
     <string name="no_chapter_text">No course content is currently available.</string>
+    <!-- Label shown when no videos are available in a course -->
+    <string name="no_videos_text">No videos are currently available.</string>
```

Comment:
Once Liz replies on the icon thing on title bar, I'll update this PR at once with the update string in this case i.e.
**This course does not include any videos.**